src/PackagingProjects/BuildAgentPayload.csproj

```diff
@@ -12,64 +12,65 @@
     <SourcesRoot>$(MSBuildThisFileFullPath)\..\..</SourcesRoot>
     <DestinationDir>$(SourcesRoot)\..\DeploymentArtifacts\BuildAgentPayload\$(Configuration)\</DestinationDir>
     <WorkDestinationDir>$(DestinationDir)Work</WorkDestinationDir>
+    <ClassicDestinationDir>$(DestinationDir)Work\$(ScannerNetFxVersion)</ClassicDestinationDir>
+    <DotnetDestinationDir>$(DestinationDir)Work\$(ScannerNetCoreAppVersion)</DotnetDestinationDir>
+
     <!-- OutputPath is required by the Clean and Rebuild tasks -->
     <OutputPath>bin\$(Configuration)\</OutputPath>
   </PropertyGroup>
 
+  <!-- Target files. -->
+  <ItemGroup>
```

Comment:
It would be worth adding a comment here to make it explicit that we expect the targets files to be identical (or perhaps just renaming the ItemGroup to _SharedTargetFiles_ would be enough).